frontend/model/state.js

```diff
@@ -243,13 +243,84 @@ const getters = {
   ourUsername (state) {
     return state.loggedIn && state.loggedIn.username
   },
+  ourUserDisplayName (state, getters) {
```

Comment:
Add todo refactor "userDisplayName"  in Profile.vue, Welcome.vue (coming with my commit, but I don't want to mess with this code)

Reply:
done